### PR TITLE
Added --output option to redirect olly printing

### PR DESCRIPTION
**Pull request by ElectreAAS**
This will come in handy for [this sandmark issue](https://github.com/ocaml-bench/sandmark/issues/358)

### CODE REVIEW

File: bin/olly.ml [hunk 1]
```diff
@@ -225,6 +227,17 @@ let () =
     Arg.(value & flag & info [ "json" ] ~docv:"json" ~doc)
   in
 
+  let output_option =
+    let doc =
+      "Redirect the output of `olly` to specified file. The output of the \
```
Comment (kayceesrk) [inline bin/olly.ml:232]: What does "output of the command is not redirected" mean?

Comment (ElectreAAS) [reply bin/olly.ml:232]: If `prog.exe` prints on stdout and/or stderr, then `olly latency -o some_file prog.exe` will print the latency info in `some_file`, but the original prints will still go to stdout/stderr

**Comment by ElectreAAS** [comment]
Just realized my shenanigans with the filename make little sense in olly, I'll stick to making a more precise script in sandmark

**Comment by ElectreAAS** [comment]
My last commit here is a partial revert, as I remove the printing of the name altogether. It was messy and in our use-case where `exec_args` contained something like `taskset ... actual_command`, only `taskset` was being printed.
The functionality has been moved in the bash script in sandmark, as per my latest commit [on the sandmark PR](https://github.com/ocaml-bench/sandmark/pull/390)

File: bin/olly.ml (hunk 1 above)
Comment (kayceesrk) [reply bin/olly.ml:227]: It might be useful to add a description in the PR message to indicate how this command works like ^^.

**Comment by kayceesrk** [comment]
Lgtm.

**Review by kayceesrk — COMMENTED** [review]
LGTM.

**Pull request merged**